rasa/core/policies/rule_policy.py

```diff
@@ -887,7 +1006,7 @@ def predict_action_probabilities(
         """Predicts the next action (see parent class for more information)."""
         (
             rules_action_name_from_text,
-            self._prediction_source,
+            prediction_source_from_text,
```

Comment:
The name seems quite confusing, what about something like:
```suggestion
            prediction_source_if_not_loop_prediction,
```

Reply:
but it is not prediction source if not loop prediction, why do you want to rename to that?

Reply:
nevermind 👍🏻